openapi_python_client/templates/types.py

```diff
@@ -15,4 +15,17 @@ def to_tuple(self) -> Tuple[Optional[str], Union[BinaryIO, TextIO], Optional[str
         return self.file_name, self.payload, self.mime_type
 
 
-__all__ = ["File"]
+T = TypeVar("T")
+
+
+@dataclass
```

Comment:
Is there a reason why this is a `dataclass` instead of an `attr.s`?

Reply:
Just haven't converted anything except the client yet. I plan to convert both generated classes and those used internally before 0.6.0 is done.